google-chrome/verify-install.sh

```diff
@@ -0,0 +1,5 @@
+#installed via construct
+#installed directly
```

Comment:
These comments aren't clear without the context of your head.  If you can't put the between two lines, explain both lines